client/app/app.less

```diff
@@ -160,6 +159,7 @@
 @import './telecom/telephony/service/fax/campaigns/read/telecom-telephony-service-fax-campaigns-read.less';
 @import './telecom/telephony/service/fax/settings/telecom-telephony-service-fax-settings.less';
 @import './telecom/telephony/telecom-telephony.less';
+@import '../components/navbar/navbar.less';
```

Comment:
We have something similar in https://github.com/ovh-ux/ovh-manager-telecom/pull/1214/files#diff-04bb1bd0b58a9a33f45862987152ac1fR127 : 
```less
@import './common/less/navbar.less';
```